src/lib/y2network/wicked/config_reader.rb

```diff
@@ -214,6 +219,10 @@ def sysctl_config_file
         @sysctl_config_file.load
         @sysctl_config_file
       end
+
+      def issues_list
```

Comment:
I am not sure if this does not cause nasty side effects. What e.g. happens if I call `config_reader.read` it reports some issues, User fix it on background and do read again so it calls again `config_reader.read` do I get also old issues list or not?

Reply:
Actually, I would use a second reader. But it is true that it could be confusing, so I have adapted that code to avoid that side effect.